src/core/checkedint.d

```diff
@@ -19,10 +19,9 @@
  * relative to the cost of the operation itself, compiler implementations are free
  * to recognize them and generate equivalent and faster code.
  *
- * References: $(LINK2 http://blog.regehr.org/archives/1139, Fast Integer Overflow Checks)
```

Comment:
Function implementations are completely rewritten so there is no connection with this blog any more. Also after looking into the blog it looks very silly: after the author performs bitshifts in `checked_add_3` instead of using bitwise operators (like done in this pull) he uses this long and slow way:
```C
(sa && sb && !sr) ||
(!sa && !sb && sr)
```

So is the really a reason to include a reference to this blog?